cypress-tests/cypress/fixtures/example.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this a placeholder or a leftover?
I would either remove it or add a comment about the purpose